cpp/modmesh/toggle/pymod/wrap_Toggle.cpp

```diff
@@ -39,6 +39,49 @@ namespace modmesh
 namespace python
 {
 
+class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSolidToggle
+    : public WrapBase<WrapSolidToggle, SolidToggle>
+{
+
+public:
+
+    using base_type = WrapBase<WrapSolidToggle, SolidToggle>;
+    using wrapped_type = typename base_type::wrapped_type;
+
+    friend root_base_type;
+
+protected:
+
+    WrapSolidToggle(pybind11::module & mod, char const * pyname, char const * pydoc);
+
+}; /* end class WrapSolidToggle */
+
+WrapSolidToggle::WrapSolidToggle(pybind11::module & mod, const char * pyname, const char * pydoc)
+    : base_type(mod, pyname, pydoc)
+{
+    namespace py = pybind11;
+
+    (*this)
+        .def(
+            "get_names",
+            [](wrapped_type const &)
+            {
+                // Hardcoding the property names in a lambda does not scale,
```

Comment:
This is just a workaround for now.  As we grow the list of solid and fixed toggles we will certainly need to improve it.